aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -37,15 +37,15 @@ Referencing ASP.NET Core in this manner is only supported for projects targeting
 
 ## Include Blazor extensibility
 
-Blazor supports WebAssembly (WASM) and Server [hosting models](xref:blazor/hosting-models). Unless there's a specific reason not to, a [Razor components](xref:blazor/components/index) library should support both hosting models. A Razor components library must use the [Microsoft.NET.Sdk.Razor SDK](xref:razor-pages/sdk).
+Blazor supports WebAssembly (WASM) and server-based [hosting models](xref:blazor/hosting-models). Unless there's a specific reason not to support both hosting models, a [Razor components](xref:blazor/components/index) library should support both hosting models. A Razor components library must use the [Microsoft.NET.Sdk.Razor SDK](xref:razor-pages/sdk).
 
 ### Support both hosting models
 
-To support Razor component consumption from both [Blazor Server](xref:blazor/hosting-models#blazor-server) and [Blazor WASM](xref:blazor/hosting-models#blazor-webassembly) projects, use the following instructions for your editor.
+To support Razor component consumption from both [Blazor Server](xref:blazor/hosting-models#blazor-server) and [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) projects, use the following instructions for your editor.
```

Comment:
Since you defined the acronym above, I recommend sticking with the acronym.

```suggestion
To support Razor component consumption from both [Blazor Server](xref:blazor/hosting-models#blazor-server) and [Blazor WASM](xref:blazor/hosting-models#blazor-webassembly) projects, use the following instructions for your editor.
```

Reply:
I didn't define the acronym ... "WebAssembly (WASM)" is the browser-based technology, while "Blazor WebAssembly" is the Blazor hosting model. This wouldn't be correct. The product name is truly "Blazor WebAssembly."

Reply:
Oh ... one more quick super-tiny nit. By convention in the Blazor docs, WebAssembly always (er .... *almost always* anyway) appears first. I'll swap the order here to match.